Package.swift

```diff
@@ -4,19 +4,25 @@ import PackageDescription
 let package = Package(
 	name: "ApertureCLI",
 	platforms: [
-		.macOS(.v10_13)
+		.macOS(.v13)
 	],
 	products: [
 		.executable(
 			name: "aperture",
 			targets: [
 				"ApertureCLI"
 			]
+		),
+		.library(
+			name: "aperture-module",
+			type: .dynamic,
+			targets: ["ApertureModule"]
 		)
 	],
 	dependencies: [
-		.package(url: "https://github.com/wulkano/Aperture", from: "2.0.1"),
-		.package(url: "https://github.com/apple/swift-argument-parser", from: "1.3.1")
+		.package(url: "https://github.com/wulkano/Aperture", branch: "george/rewrite-in-screen-capture-kit"),
```

Comment:
The branch here should change back to the real version when the Aperture PR is merged/released